base/index.html

```diff
@@ -15,6 +15,7 @@
             <div class="w-50 h-100 flex flex-column">
                 <div class="pa2">
                     <select v-model="selection">
+                        <option value="basic">Base Standard GL</option>
```

Comment:
Hmm, `Base Basic GL` doesn't really roll off the tongue does it? But the `value` should match the name... so consider the awkward wording or coming up with a new name for your color schema (besides `BASIC`)?